pcdsdevices/tests/test_state.py

```diff
@@ -96,6 +96,7 @@ def test_pvstate_positioner_logic():
     lim_obj.states_enum['Unknown']
     with pytest.raises(KeyError):
         lim_obj.states_enum['defer']
+    lim_obj.destroy()
```

Comment:
What exactly is the new dangling thing to worry about here? Do we need to worry about this in the live env?

Reply:
Live environment this is not really a concern as we create signals and use them once.

For testing and ephemeral usage (typhos?) we absolutely should be more concerned about correctly destroying devices and signals.